src/openlcb/SNIPClient.hxx

```diff
@@ -88,6 +89,17 @@ public:
     {
     }
 
+    /// Flushes the pending timed operations.
```

Comment:
Is this shutdown method only for tests? Should it have an ifdef GTEST wrapper?

Reply:
Unfortunately I can't make this test only.
I have another component which has ownership of a SNIPClient. That component must call this function in its destructor. The destructor cannot be made testonly, even if in practice it is only run in tests.